js/js-stub/build.gradle.kts

```diff
@@ -2,10 +2,6 @@
  * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
-tasks.named<KotlinCompile>("compileKotlin") {
-    kotlinOptions {
-        freeCompilerArgs += "-Xallow-kotlin-package"
-    }
+plugins {
+    `stub-convention`
```

Comment:
Please rename it to `stdlib-stub-convention`, just "stub" is too broad term

Reply:
Done